.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     timeout-minutes: 20
     strategy:
       matrix:
-        os: [ubuntu-20.04, macos-11]
+        os: [ubuntu-20.04, ubuntu-22.04, macos-11]
```

Comment:
Double check error output before merging.

Reply:
And make the 22-04 check mandatory

Reply:
And separate the perf output.
And update perf to have a minimum of measurements.